sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/models/ChatRequestAssistantMessage.java

```diff
@@ -244,7 +287,7 @@ private ChatRequestAssistantMessage(BinaryData content) {
     public ChatRequestAssistantMessage(String content) {
         this.content = content == null ? null : BinaryData.fromString(content);
         this.stringContent = content;
-        this.chatMessageContentItem = null;
+        this.chatMessageContentItems = null;
```

Comment:
We have `ChatRequestAssistantMessage(String content)` and `ChatRequestAssistantMessage(List<ChatMessageContentItem> content)`, we can also add ChatRequestAssistantMessage(ChatMessageContentItem... content)`